contrib/publish-armhf.sh

```diff
@@ -11,17 +11,35 @@ HERE=`pwd`
 
 #fi
 
+get_repo_name() {
+    if  [ "openfaas-incubator/faas-idler" = $1 ]; then
+        echo "openfaas/faas-idler"
+    elif  [ "openfaas/faas" = $1 ]; then
+        echo "openfaas/gateway"
+    elif  [ "openfaas/nats-queue-worker" = $1 ]; then
+        echo "openfaas/queue-worker"
+    else
+        echo $1
+    fi
+}
+
 for i in "${repos[@]}"
 do
    cd $HERE
 
-   echo "$i"
+   echo -e "\nBuilding: $i\n"
    git clone https://github.com/$i ./staging/$i
    cd ./staging/$i
    pwd
    export TAG=$(git describe --abbrev=0 --tags)
    echo "Latest release: $TAG"
 
-   make ci-armhf-build ci-armhf-push
+   REPOSITORY=$(get_repo_name $i)
+   TAG_PRESENT=$(curl -s "https://hub.docker.com/v2/repositories/${REPOSITORY}/tags/${TAG}-armhf/" | jq -r ".detail")
```

Comment:
Can we do this without a dependency on `jq`?

Reply:
Sure, I'll try to come up with an approach without any non-native dependencies :+1: